disco/voice/client.py

```diff
@@ -182,6 +270,69 @@ def send(self, op, data):
             'd': data,
         }), self.encoder.OPCODE)
 
+    def pipe_voice_into_file(self, member, file_object, buffer_size=0.2):
```

Comment:
Realistically we should build out a similiar framework to what voice sending has, allowing users to pipe this around and give a few common things;

- muxing together voice
- outputing voice to a file
- retransmitting voice to discord

I think for this PR it would be ok if you just nix this entirely and we work on this stuff after the fact.